doc/server_API.md

```diff
@@ -188,14 +174,14 @@ On success:
 
 
 
-## add_to_whitelist
+## add_plate_to_whitelist
 ### Call
-``` POST /add_to_whitelist/ ```
+``` POST /add_plate_to_whitelist/ ```
 ### Body
 
 ```json
 {
-  "whitelist_id":12345,
+  "whitelist_unique_name":12345,
```

Comment:
I don't think it is neccesary to underline that whitelist name is unique - simply `whitelist_name` will suffice.

Also add error code for situation when plate is wrongly formatted